zwave_js_server/firmware.py

```diff
@@ -17,18 +19,46 @@ async def begin_firmware_update(
     file_format: Optional[str] = None,
 ) -> None:
     """Send beginFirmwareUpdate command to Node."""
-    client = Client(url, session)
-    await client.connect()
-    await client.set_api_schema()
+    client = await session.ws_connect(url, compress=15, max_msg_size=0)
```

Comment:
We're missing some exception handling and error raising here and around the received message and for invalid server version when not using the client.